<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMenu</class>
 <widget class="QDialog" name="AddMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>250</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>400</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="labelTimerName">
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>36</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="whatsThis">
           <string comment="max 36 symols"/>
          </property>
          <property name="text">
           <string>Тип таймера</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QFrame" name="frame">
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="alarmButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <family>Montserrat</family>
                 <pointsize>24</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Будильник</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="timerButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <family>Montserrat</family>
                 <pointsize>24</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Відлік</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QFrame" name="additionalSettings">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="additionalSettingsText">
             <property name="font">
              <font>
               <family>Montserrat</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Додаткові можливості:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignHCenter|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="executeProgram">
             <property name="font">
              <font>
               <family>Montserrat</family>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="text">
              <string>    Запуск програми</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="executeProgramWidget" native="true">
             <property name="font">
              <font>
               <family>Montserrat</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLineEdit" name="executeProgramPath">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Montserrat</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="alignment">
                 <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="findProgramButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Montserrat</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Знайти</string>
                </property>
                <property name="autoRepeatInterval">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="openDocument">
             <property name="font">
              <font>
               <family>Montserrat</family>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="text">
              <string>    Відкриття документу</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="openDocumentWidget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLineEdit" name="openDocumentPath">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Montserrat</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="findDocumentButton">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Montserrat</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Знайти</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="backButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Назад</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
